src/BookReader.js

```diff
@@ -662,87 +671,140 @@ BookReader.prototype.resize = function() {
 };
 
 /**
- * Binds keyboard event listeners
+ * Returns true if passed element activates or maintains BookReader keyboard shortcut focus
+ * @param {HTMLElement} element
+ * @returns {boolean}
  */
-BookReader.prototype.setupKeyListeners = function() {
-  var self = this;
+BookReader.prototype.isKeyFocus = function (element) {
+  // BookReader and ItemNavigator
+  let selectors = this.el;
+  selectors += ',' + 'item-navigator';
```

Comment:
Are these all elements from petabox? Just wondering if it's possible to not hard code these since it kind of binds this to the petabox structure, but it may not be.

Reply:
The BookReader container structure is in flux. I expect this to change -- and tried to isolate all the changes here, but we're going to have to monitor it.

It still works outside of petabox.